florist/api/client.py

```diff
@@ -1,11 +1,53 @@
 """FLorist client FastAPI endpoints."""
+import uuid
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+import torch
 from fastapi import FastAPI
 from fastapi.responses import JSONResponse
+from fl4health.clients.basic_client import BasicClient
+
+from florist.api.clients.mnist import MnistClient
+from florist.api.launchers.local import launch_client
+from florist.api.monitoring.metrics import RedisMetricsReporter
+
 
+LOG_FOLDER = Path("logs/client/")
 
 app = FastAPI()
 
 
+class Clients(Enum):
```

Comment:
Is it best to leave all definitions besides endpoints in seperate files and import? Or does it not really matter. I don't have a lot of experience with writing APIs so if this is an unnecessary nit, feel free to ignore

Reply:
That makes sense, will move.